### PR TITLE
Regex fix

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45755

### CODE REVIEW

**Comment by Shellyber** [comment]
@iyeshaya Seems like tests are failing. Please have a look

File: Packs/CommonTypes/ReleaseNotes/3_2_3.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Types
+- **urlRep**: Fixed an issue where regex failed to parse url addresses.
```
Comment (Shellyber) [inline Packs/CommonTypes/ReleaseNotes/3_2_3.md:3]: Let's elaborate on the fixed introduced here, like the support for ipv6 and the rest.

File: Packs/CommonTypes/ReleaseNotes/3_2_3.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Indicator Types
+- **urlRep**: Updated the regex relevant catching URL addresses:
```
Comment (Shellyber) [inline Packs/CommonTypes/ReleaseNotes/3_2_3.md:3]:
```suggestion
- **urlRep**: Updated the URL indicator regex with the following:
```

File: Packs/CommonTypes/ReleaseNotes/3_2_3.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    - Added support to IPV6.
+    - Fixed an issue where the regex would not catch valid URLs.
```
Comment (Shellyber) [inline Packs/CommonTypes/ReleaseNotes/3_2_3.md:5]:
```suggestion
    - Fixed an issue where the regex did not catch valid URLs.
```

**Review by Shellyber — COMMENTED** [review]
Don't forget to update the tpb with the relevant changes

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**